src/main/java/com/google/sps/servlets/SearchServlet.java

```diff
@@ -44,9 +45,11 @@
 
 @WebServlet("/search")
 public class SearchServlet extends HttpServlet {
+  // The number of recipes to be displayed on the search list.
+  private static final int RECIPES_LIMIT = 10;
 
   /**
-   * Search and returns a list of first 10 recipes with the title matching the given parameter(searchterm).
+   * Search and returns a list of first 10 recipes with the title matching the given parameters(searchterm, difficulty & time).
```

Comment:
the given parameters: searchterm, difficulty, and time.

Please write the comments following the correct grammar rules!